ALLHUBS-Jan-2024-Liftoff/code-stars
# 9 review view

In this branch, I:
- Created the DisplayReview component, which takes a review id as props to display review info
- Created the DisplayAllReviews component, which takes a campground id as props and displays all related reviews
- Created the DisplayRating component, which takes an integer as props and returns stars (wrong branch, oops)
- Styled the reviews with Bootstrap
- Renamed the review folder for better organization

To test it:
- Start both servers
- Make sure you have campgrounds and reviews in your database. Reviews must have non-null campground_id column! If they don't, create some new reviews using the form.
- Navigate to the search page, and click "view" next to any campground
- Verify that the related reviews render at the bottom of the page
